### PR TITLE
entrypoint override

### PR DESCRIPTION
**Pull request by mhald**

### CODE REVIEW

File: docs/packaging-an-application/components-and-containers.md [hunk 1]
```diff
@@ -371,4 +371,12 @@ Add extra hostname mappings.  See [extra_hosts](https://docs.docker.com/compose/
   - "redis:10.0.1.32"
 ```
 
+### Entrypoint
+For third party containers the entrypoint the container comes with may be too strict.  To include such containers the entrypoint can be overwritten using exec entrypoint form which takes an list.
```
Comment (emosbaugh) [inline docs/packaging-an-application/components-and-containers.md]: why dont you revise this sentence and ill look again.

**Review by emosbaugh — CHANGES_REQUESTED** [review]
please review

File: docs/packaging-an-application/components-and-containers.md [hunk 2]
```diff
@@ -371,4 +371,12 @@ Add extra hostname mappings.  See [extra_hosts](https://docs.docker.com/compose/
   - "redis:10.0.1.32"
 ```
 
+### Entrypoint
+When working with third party containers you may find that the provided entry point is too strict.  For those
+cases use the entrypoint option to override the provided container entry point.
```
Comment (emosbaugh) [inline docs/packaging-an-application/components-and-containers.md]: i would just mention that it expects an array

**Review by emosbaugh — CHANGES_REQUESTED** [review]
changes requested

**Review by emosbaugh — APPROVED** [review]

File: docs/packaging-an-application/components-and-containers.md (hunk 2 above)
Comment (marccampbell) [inline docs/packaging-an-application/components-and-containers.md]: I don't understand "entry point is too strict".  This feature is useful for other use cases, such as reusing an image but not running the executable in the entrypoint, right?  Should this sentence remain?

**Pull request merged**